dimension_reduction/pca_example.Rmd

```diff
@@ -0,0 +1,117 @@
+---
+title: "PCA"
+author: "ALSF CCDL - Candace Savonen"
+output:   
+  html_notebook: 
+    toc: true
+    toc_float: true
+---
+
+*Purpose*: This notebook illustrates how to perform Principal Components 
+Analysis and plot the scores using ggplot2. 
+
+Recommended articles on PCA: 
+- [Overall Explanation by Matt Brems](https://towardsdatascience.com/a-one-stop-shop-for-principal-component-analysis-5582fb7e0a9c)
```

Comment:
I think you need a new line before this one to get this to render properly as a bulleted list.